test/subscription_map.hpp

```diff
@@ -438,32 +431,49 @@ class multiple_subscription_map
 
     // Insert a key => value at the specified subscription path
     // returns the handle and true if key was inserted, false if key was updated
-    template <typename V>
-    std::pair<handle, bool> insert_or_update(MQTT_NS::string_view subscription, Key const &key, V&& value) {
+    template <typename K, typename V>
+    std::pair<handle, bool> insert_or_update(MQTT_NS::string_view subscription, K&& key, V&& value) {
```

Comment:
Rename to insert_or_assign ?

Reply:
Yes, please.

Reply:
Thank you, i will do one more push and include fix for #714 

lookup will return optional<handle>, returning an empty optional when subscription was not found. (Instead of crashing now)

Reply:
But can I push somehow, without triggering your CI ?